### DISCUSSION TITLE
Impact of changing DKIM_SELECTOR

### DISCUSSION THREAD

**Discussion by dVerhees** [Mailu/Mailu#3264] [User support]
Hi,

I've been using mailu for some time now without setting the DKIM_SELECTOR, which works great, but we're looking into changing it to another value to avoid conflicts and to make it more clear what the DKIM selector is for. 

How will this impact existing keys? Will these instantly become invalid and be removed? Will rspamd ignore the existing keys and need me generate new ones?
Is there a way to set a selector for only new or specific domains?

**Comment by dVerhees** [comment]
Any chance for a response on this topic?

**Reply by nextgens** [reply]
DKIM private keys are used when emails are sent. If you change the selector you will need to rekey unless you rename the key file... and to update DNS records accordingly.

Whether you do or you don't reuse the same key has no impact on existing email signatures. As long as you host the DNS entry for the old selector (which is the public key) these will remain verifiable.

**Reply by nextgens** [reply]
And you probably do not care about the verifiability of emails that have already been delivered.
